packages/bsb-core/bsb/config/_attrs.py

```diff
@@ -1051,7 +1165,9 @@ def __populate__(self, instance, value, unique_list=False):
         should_pop = has_pop and (not unique_list or is_new)
         should_ref = should_pop or not has_pop
         if should_pop:
-            population.append(value)
+            # use list.append as cfglist.append will call resolve_reference
+            # creating an infinite loop
+            builtins.list.append(population, value)
```

Comment:
Pfffff I only vaguely remember this feature ... What does `populate` do again? And there was a related concept of "back referencing" or "back populating" or something?

Reply:
Kinda yes, this feature is quite weird. So basically, it is a reversed reference attribute system where the reference target is filled with the value of the reference attribute. Frankly I did not understood fully what is the difference with the back reference system but it is used for the `Region`